src/Products/PlonePAS/profiles.zcml

```diff
@@ -10,5 +10,31 @@
       description="Extension profile for default PlonePAS setup."
       provides="Products.GenericSetup.interfaces.EXTENSION"
       />
+  <genericsetup:importStep
+      description="Configure PlonePas"
+      handler="Products.PlonePAS.setuphandlers.setupPlonePAS"
+      name="plonepas"
+      title="PlonePAS setup">
+    <depends name="componentregistry" />
+    <depends name="controlpanel" />
+    <depends name="memberdata-properties" />
+    <depends name="rolemap" />
+  </genericsetup:importStep>
+
+  <genericsetup:registerProfile
+      name="root-cookie"
+      title="Zope Root Cookie Login"
+      description="Change the Zope root `/acl_users` to use a simple cookie login form
+		   instead of HTTP `Basic ...` for authentication."
+      provides="Products.GenericSetup.interfaces.EXTENSION"
+      />
+  <genericsetup:importStep
```

Comment:
Could you replace the import step with a `post_handler` in the `registerProfile` directive? Then you don't need the `plone-pas-zope-root-cookie.txt` flag file.

Reply:
Oh thank god, I always *hated* that pattern.  Can you point me to an example and optionally docs?  I would *love* to make that change!

Reply:
Example in `plone.app.contenttypes`:
https://github.com/plone/plone.app.contenttypes/blob/eed0d95555fc5e3e85bf7c7dab2d350f213c7ad4/plone/app/contenttypes/profiles.zcml#L12
It even has two in this same file.

Documentation: not available, except in the `CHANGES.rst` of `GenericSetup` 1.8.2. This is my PR from 2016 where I added support for this handler:
https://github.com/zopefoundation/Products.GenericSetup/pull/24
You can also do a `pre_handler`, but this is not used much. In Plone core I only see this in an [uninstall profile in `CMFPlacefulWorkflow`](https://github.com/plone/Products.CMFPlacefulWorkflow/blob/00af974ce2386f1b8aeb79d76a5f53bed7f563e7/Products/CMFPlacefulWorkflow/profiles.zcml#L28).

Reply:
[Pushed](https://github.com/plone/Products.PlonePAS/pull/66/commits/22ba99f7ee9074d6cbbab0ec680eb4582ef4efce).  So much better!